.github/workflows/pull-request-label.yml

```diff
@@ -0,0 +1,166 @@
+name: Pull Request Maintenance
+on:
+  pull_request_target:
+    # make sure that when the PR changes, we also update
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  conventional-commit-labeler:
```

Comment:
Hmm I could swear I saw this one before :P

Reply:
Yep you should have. :P :D

Reply:
For the record, this is a modified copy of https://github.com/open-component-model/open-component-model/blob/main/.github/workflows/pull-request.yaml since I like the way that labeler works.